script/outside.js

```diff
@@ -333,6 +326,16 @@ var Outside = {
 		}
 	},
 	
+	workers_call: function(i){
+		var child = $(this);
+		var cName = child.children('.row_key').text();
+		if(cName != 'gatherer') {
+			if(cName < lk) {
+				curPrev = child.attr('id');
```

Comment:
`curPrev` is undefined here, as it was previously coming from the closure scope of the inline function.